src/cli/mod.rs

```diff
@@ -5,7 +5,7 @@ mod deploy;
 #[cfg(enarx_with_shim)]
 mod key;
 mod package;
-mod platform;
+pub(crate) mod platform;
```

Comment:
Instead of doing this, move `caching` module to top-level and make it public, then you can import it in `backend`, `cli` and the tests

Reply:
Done